examples/injection/module/rust/src/lib.rs

```diff
@@ -66,7 +66,7 @@ mod proto {
 
 use oak::{
     grpc,
-    io::{Receiver, Sender},
+    io::{Receiver, ReceiverExt, Sender, SenderExt},
```

Comment:
I imagine eventually `oak::io::Receiver` and `oak::io::Sender` would be replaced by simpler objects (possibly with same name) defined in `oak_abi` , and also generated by our custom prost logic, is that correct?

Reply:
Yes, although it would probably move to the new `oak_io` crate as part of #1324.